packages/vscode-extension-bpmn-editor/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "vscode-extension-bpmn-editor",
+  "description": "",
+  "version": "0.4.2",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kiegroup/kie-tooling-store.git"
+  },
+  "scripts": {
+    "build-ext": "rm -rf dist && webpack --mode production --devtool none",
+    "pack-ext": "vsce package --yarn -o ./dist/vscode_extension_bpmn_editor_$npm_package_version.vsix",
+    "pack": "yarn build-ext && yarn pack-ext",
+    "compile": "yarn build-ext",
+    "watch": "yarn build-ext"
+  },
+  "dependencies": {
+    "@kogito-tooling/kie-bc-editors": "0.4.2",
+    "@kogito-tooling/microeditor-envelope": "0.4.2",
+    "@kogito-tooling/vscode-extension": "0.4.2",
```

Comment:
We should discuss what's the best first version for the `@kogito-tooling/*` packages.

Reply:
For master, I think it's fine to keep the same as the community. As soon as we have a position for productization, we will need to branch/tag something and we change those properties accordingly.

Reply:
/cc @porcelli